src/components/TodoList/TodoList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Todo from '../Todo/Todo';
+import './TodoList.css';
+
+const TodoList = ({ todos }) => (
+  todos.map(todo => (
+    <ul className="list">
+      <li key={todo.id} className="list__item">
+        <Todo todo={todo} />
```

Comment:
You can use spread operator here and then easily destructure `todo` inside `Todo` component.
```suggestion
        <Todo todo={...todo} />
```